tests/acceptance/helpers.py

```diff
@@ -149,17 +150,7 @@ def activate_experiment(sess):
     payload = {"userId": "matjaz", "userAttributes": {"attr_1": "hola"}}
     params = {"experimentKey": 'ab_test1'}
 
-    request, request_result = create_and_validate_request(ENDPOINT_ACTIVATE, 'post', payload, params)
-
-    # raise errors if request invalid
-    request_result.raise_for_errors()
-
-    resp = sess.post(BASE_URL + ENDPOINT_ACTIVATE, params=params, json=payload)
-
-    response_result = create_and_validate_response(request, resp)
-
-    # raise errors if response invalid
-    response_result.raise_for_errors()
+    resp = create_and_validate_request_and_response(ENDPOINT_OVERRIDE, 'post', sess, str(payload), list(params))
```

Comment:
there is json deserializing issue with payload, the create_and_validate_request_and_response function has json.loads(payload) and this doesn't support the payload of activate_experiment function. Tried googling the solution, need help with this.